Examples/Scripts/SetUpSPECIES.sh

```diff
@@ -0,0 +1,363 @@
+#! /bin/bash
+
+# Species specific config file 
+
+# Takuya Hayashi, RIKEN Japan
+# Akiko Uematsu, RIKEN, Japan
+# Yuki Hori, RIKEN Japan
+# Chad Donahue, Washington University St. Louis, USA
+# Matthew F Glasser, Washington University St. Louis, USA
+# Copyright (c) 2016-2023
+# All rights reserved.
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"         # Debugging functions; also sources log.shlib
```

Comment:
Don't imply that logging isn't sourced by newopts.
```suggestion
source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"     #also sources log.shlib
source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
```

Reply:
Thank you for the suggested fix!
a1fb694c04a20c39b2beeba12ba42d8976b4f47e